freeipa.spec.in

```diff
@@ -70,10 +69,9 @@
 
 # fix for segfault in python3-ldap, https://pagure.io/freeipa/issue/7324
 %global python_ldap_version 3.1.0-1
-# python3-lib389
-# Fix for "Installation fails: Replica Busy"
-# https://pagure.io/389-ds-base/issue/49818
-%global ds_version 1.4.0.16
+# Fix for create suffix
+# https://pagure.io/389-ds-base/issue/49984
+%global ds_version 1.4.0.21
```

Comment:
389-ds 1.4.0.21 will also fix issue [7517](https://pagure.io/freeipa/issue/7517), but 1.4.1.1 will be required for Fedora 30.

Reply:
Fedora >= 30 now require 1.4.1.1